src/features/user/Account/EditModal.tsx

```diff
@@ -118,27 +106,29 @@ export const EditModal = ({
     if (
       new Date(data.currentPeriodEnd).toDateString() ==
         new Date(record.currentPeriodEnd).toDateString() ||
-      bodyToSend.nextBilling === null
+      payload.nextBilling === null
     ) {
-      delete bodyToSend.nextBilling;
+      delete payload.nextBilling;
     }
     if (data.frequency.toLowerCase() === record.frequency) {
-      delete bodyToSend.frequency;
+      delete payload.frequency;
     }
     if (data.amount == record.amount) {
-      delete bodyToSend.centAmount;
+      delete payload.centAmount;
     }
 
-    if (Object.keys(bodyToSend).length !== 0) {
+    if (Object.keys(payload).length !== 0) {
       try {
-        const res = await putAuthenticatedRequest<ModifyDonations>({
-          tenant: tenantConfig?.id,
-          url: `/app/subscriptions/${record?.id}?scope=modify`,
-          data: bodyToSend,
-          token,
-          logoutUser,
+        const res = await putApiAuthenticated<
+          Subscription,
+          ModifySubscriptionApiPayload
+        >(`/app/subscriptions/${record?.id}`, {
+          queryParams: { scope: 'modify' },
+          payload,
         });
```

Comment:
There's a warning on line 132 as `response` does not exist within `res`

`window.open(res.response.confirmationUrl, '_blank');`